providers/keycloak/docs/auth-manager/manage/permissions.rst

```diff
@@ -19,4 +19,72 @@
 Manage user permissions with Keycloak auth manager
 ==================================================
 
-To be completed
+Please setup your Keycloak instance and required parameters for the Keycloak auth manager.
+
+Setting up the permissions can easily done using CLI commands.
+It can create the permissions and needed resources easily.
+
+There is two options to create the permissions:
+
+* Create all permissions in one go using single CLI command
+* Create step by step using the CLI commands
```

Comment:
```suggestion
* Create step-by-step using the CLI commands
```

What is this line supposed to mean? It creates a step-by-step (of what?) for the user??